### PR TITLE
Update to SDK 15.6.5

### PR DESCRIPTION
**Pull request by duclong201**
[Jira ticket](https://bluedotinnovation.atlassian.net/jira/software/c/projects/BD/boards/42?modal=detail&selectedIssue=BD-4470)

### CODE REVIEW

File: Binding/BDPointSDKApiDefinitions.cs [hunk 1]
```diff
@@ -395,16 +400,114 @@ interface BDPGeometry : IBDPDeepCopy
...
+*/[Protocol]
+[BaseType (typeof(NSObject))]
+interface BDPJSONSerializable
+{
+	// @required -(NSDictionary * _Nullable)dictionaryRepresentation;
+	[Abstract]
+	[NullAllowed, Export ("dictionaryRepresentation")]
+	[Verify (MethodToProperty)]
+	NSDictionary DictionaryRepresentation { get; }
+}
```
Comment (datkinnguyen) [inline Binding/BDPointSDKApiDefinitions.cs:429]: is all this auto-generated?

File: README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Below are basic steps to build a wrapper. Currently it uses `Xamarin.Swift4` NuG
 
     This will update bindings in the `/Bindings` folder
 
-3. `git diff` bindings with the previous version. If there are significunt changes in the API, it will be visible in the diff and project classes will require updating. Otherwise, project classes remain the same.
+3. `git diff` bindings with the previous version. If there are significant changes in the API, it will be visible in the diff and project classes will require updating. Otherwise, project classes remain the same.
```
Comment (datkinnguyen) [inline README.md:26]: Could you add some guide to how to actually release it to the public?

**Review by datkinnguyen — CHANGES_REQUESTED** [review]
Could you add guideline to Release part? Thanks!

File: Binding/BDPointSDKApiDefinitions.cs (hunk 1 above)
Comment (duclong201) [reply Binding/BDPointSDKApiDefinitions.cs:429]: Yes, these are all auto-generated.

File: README.md (hunk 2 above)
Comment (duclong201) [reply README.md:26]: Sure

File: Binding/BDPointSDKApiDefinitions.cs (hunk 1 above)
Comment (datkinnguyen) [reply Binding/BDPointSDKApiDefinitions.cs:429]: nice!

**Comment by duclong201** [comment]
ReleaseNotes updated.

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Created by Bluedot Innovation in 2023.
+Copyright © 2023 Bluedot Innovation. All rights reserved.
+
+By downloading or using the iOS Bluedot Point SDK for Xamarin, You agree to the Bluedot Terms and Conditions (http://www.bluedotinnovation.com/html/downloads/pdfs/terms-and-conditions-bluedot-070814.pdf)
+and Privacy Policy (http://www.bluedotinnovation.com/html/downloads/pdfs/privacy-policy-bluedot-170815.pdf)
+and Billing Policy(http://www.bluedotinnovation.com/html/downloads/pdfs/privacy-policy-bluedot-170815.pdf)
```
Comment (datkinnguyen) [inline LICENSE.md]: This link doesn't work. Can you update it with a working one from Dan?

Comment (duclong201) [reply LICENSE.md]: Just updated the urls

**Review by datkinnguyen — APPROVED** [review]

**Pull request merged**